src/Files.App/Filesystem/StorageItems/ZipStorageFile.cs

```diff
@@ -1,5 +1,6 @@
 using Files.App.Extensions;
 using Files.App.Helpers;
+using Files.Backend.Helpers;
```

Comment:
Won't tag all of them, but I suppose they have a use?

Reply:
This PR changes a namespace from `Files.App.Helpers` to `Files.Backend.Helpers`